core-chart/templates/_helpers.tpl

```diff
@@ -18,10 +22,10 @@ Expand ITS PCH custeradm container.
 {{- end }}
 
 {{/*
-Expand ITS PCH statusaddon container.
+Container for installing the OCM status add-on using Helm.
 */}}
 {{- define "pch.its.statusaddon" -}}

```

Comment:
Since this is now used in only one place, it could be inlined there rather than defined here, right?